src/main/xar-resources/data/upgrading.xml

```diff
@@ -121,13 +121,13 @@
                     <tbody>
                 <tr>
                     <td>3.6</td>
-                    <td><ulink url="http://exist-db.org/exist/apps/wiki/blogs/eXist/">release notes</ulink></td> <!--needs update with next breaking release DP-->
+                    <td><ulink url="http://exist-db.org/exist/apps/wiki/blogs/eXist/eXistdb360">release notes</ulink></td> <!--needs update with next breaking release DP-->
```

Comment:
I think you would be better to use this URL - http://exist-db.org/exist/apps/wiki/blogs/eXist/search.html?field=tags&q=release

Then we don't need to remember to update it for every release.